workflows/differentialabundance.nf

```diff
@@ -214,7 +232,12 @@ workflow DIFFERENTIALABUNDANCE {
         ch_raw = ch_validated_assays.raw
         ch_norm = ch_validated_assays.normalised
         ch_matrix_for_differential = ch_norm
-    } else{
+    }
+    else if (params.study_type == 'geo_soft_file') {
+        ch_norm = VALIDATOR.out.assays
+        ch_matrix_for_differential = ch_norm
```

Comment:
suggest changing the logic to set `ch_matrix_for_differential = ch_norm` for all non-rnaseq.